text/0000-ember-data-javascript-module-api.md

```diff
@@ -0,0 +1,130 @@
+- Start Date: 2017-07-25
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+Align Ember Data module API with [RFC #176](https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md).
+
+# Motivation
+
+This document proposes changes to the modules exported by Ember Data to make it consistent with the changes to the Ember module API proposed in [RFC #176](https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md).
+
+# Detailed design
+
+Ember Data will stick with 1 top level namespace of `@ember-data`. Under it there are 6 nested module 
+namespaces where exposed components could live. Someday these namespaces could be moved into their own 
+top level scoped package (for example `@ember-data/serializer` could become `@ember-data/serializer`) however,
```

Comment:
what's the difference between `@ember-data/serializer` and `@ember-data/serializer`?